### DISCUSSION TITLE
How to update the pipeline to the latest version v2.0.0-alpha.3

### DISCUSSION THREAD

**Discussion by h171e** [PGScatalog/pgsc_calc#215] [Q&A]
I don't know how to update to v2.0.0-alpha3

Tried with `nextflow run pgscatalog/pgsc_calc -latest` which results in this message 
```
Pulling pgscatalog/pgsc_calc ...
pgscatalog/pgsc_calc contains uncommitted changes -- cannot pull from repository
```
Deleted the **pgscatalog** folder within **.nextflow** and ran a test but it looks like it downloads alpha 2. 
```
------------------------------------------------------
  pgscatalog/pgsc_calc v2.0.0-alpha.2
------------------------------------------------------
```

Deleted the **pgscatalog** folder again and downloaded the **Source code (zip)** file located here: 
https://github.com/PGScatalog/pgsc_calc/releases/tag/v2.0.0-alpha.3

Placed the contents in a new .nextflow/assets/pgscataloc/pgsc_calc folder and running:
`nextflow run pgscatalog/pgsc_calc -profile docker,test` 
results in:
`Can't find git repository config file -- Repository may be corrupted: /.nextflow/assets/pgscatalog/pgsc_calc`

Deleted the **pgscatalog** folder, ran the test again to download the alpha2 version, replaced the files in the folder with the files included in the Source code (zip) file for alpha3, but running the test command indicates:
```
------------------------------------------------------
  pgscatalog/pgsc_calc v2.0.0-alpha.2
------------------------------------------------------
```

**Comment by nebfield** [comment]
We [forgot to update the version string](https://github.com/PGScatalog/pgsc_calc/blob/main/nextflow.config#L211) when we published the most recent patch 😬 Sorry!

To guarantee you're running `v2.0.0-alpha.3`, run:

```
$ nextflow pull pgscatalog/pgsc_calc -r v2.0.0-alpha.3
```

The output should match:

```
Checking pgscatalog/pgsc_calc ...
 done - revision: ddb19b3f0e [v2.0.0-alpha.3]
```

You can double check our release page and compare the [hash `ddb19b3`](https://github.com/PGScatalog/pgsc_calc/releases/tag/v2.0.0-alpha.3).

(The terminal output will still say the incorrect version until we fix it in a future release)
